lusSTR/wrappers/convert.py

```diff
@@ -101,9 +98,8 @@ def format_table(input, software, kit="forenseq"):
             ]
             flanks_list.append(flank_summary)
             continue
-
         marker = STRMarkerObject(locus, sequence, software, kit=kit)
-        if locus == "D12S391" and kit == "powerseq":
+        if locus == "D12S391" and kit == "powerseq" and software == "straitrazor":
             if "." in str(marker.canonical):
                 check_sr += 1
                 if check_sr > 10:
```

Comment:
Looking through the code I realized if I was going to be running a check on strait razor data I should make sure that strait razor was actually used (and not gene marker).